symphony-bdk-http/symphony-bdk-http-webclient/src/main/java/com/symphony/bdk/http/webclient/ApiClientWebClient.java

```diff
@@ -354,7 +368,7 @@ public String selectHeaderContentType(String[] contentTypes) {
   @Override
   public String escapeString(String str) {
     try {
-      return URLEncoder.encode(str, "utf8").replaceAll("\\+", "%20");
+      return URLEncoder.encode(str, "utf8").replace("\\+", "%20");
```

Comment:
Actually I tested it with send message to a stream that has + in it
And it gets replace with %20 and the % gets replaced by the uri builder in spring webclient